### ISSUE TITLE
Weird entries (sub-folders) in display of folders

### ISSUE THREAD

**Issue by zWhdmB5T** [flathub/com.xnview.XnViewMP#61]
Dear @gombosg @hfiguiere all,

I experience weird entries (sub-folders) in display of folders. See screenshots for details. I marked in green what I would expect as normal ("gunnersson" is my standard user on that system…).

Details:
* openSUSE Leap 15.6
* XnView MP 1.7.2 (as Flatpak, from here)

Screenshots with details:
![Bildschirmfoto_2024-07-20_20-49-17](https://github.com/user-attachments/assets/e0a3356e-1a80-4cdb-889c-1ff4803ee12c)
![Bildschirmfoto_2024-07-20_20-49-43](https://github.com/user-attachments/assets/f1fcfab4-8f2b-4eae-ba9b-c6c804bb8df1)

Any idea by what setting to avoid this?

**Comment by gombosg** [comment]
Hey, what you see there as "Computer" is not the real machine but the Flatpak sandbox root. Normally the app only has home  folder access, so the Computer doesn't make a lot of sense. The rest of the folders probably depend on the OS's Flatpak implementation and the installed Flatpak apps.

**Comment by zWhdmB5T** [comment]
Oh well, yes: "Computer" is not the "usual" "Computer" but the Flatpak sandbox root as you said.

Then, only "Arbeitsplatz" and "gunnersson" (both link to /home/gunnersson, my standard user) make sense. The other ones are unnecessary and confusing.

Is it possible to disable them?

**Comment by gombosg** [comment]
I don't think so, as long as the app displays the file system root, it will display the sandbox root, I wouldn't consider this to be a bug.

**Comment by zWhdmB5T** [comment]
When you choose "Open" from the the main menu: "File", you get what I actually would really expect…

![Bildschirmfoto_2024-07-23_22-09-03](https://github.com/user-attachments/assets/d2570877-fcf3-43b4-b18d-8ce88e8a121e)

Not all the system related folders, which are unnecessary and confusing.

**Comment by bbhtt** [comment]
Why are you pinging me? I don't maintain this app.

**Comment by gombosg** [comment]
Are you sure you haven't modified Flatpak permissions via Flatseal for example? Ideally you'd have 2 entries, Computer (flatpak sandbox) and Home (your actual home folder)

I'd start with resetting all settings, [uninstalling](https://docs.flathub.org/docs/for-users/uninstallation) and deleting all data. Then reinstall with default settings to see if the issue persists.

![Image](https://github.com/user-attachments/assets/5c58a54a-a49f-41a8-91cc-267bc500a2bd)

**Comment by gombosg** [comment]
Great news! Good to know that the author is responsive to these Linux issues.
